src/applications/claims-status/tests/e2e/01.claim-status.v2.cypress.spec.js

```diff
@@ -0,0 +1,82 @@
+import TrackClaimsPageV2 from './page-objects/TrackClaimsPageV2';
+import claimsList from './fixtures/mocks/lighthouse/claims-list.json';
+import claimDetails from './fixtures/mocks/lighthouse/claim-detail-closed.json';
+import claimDetailsOpen from './fixtures/mocks/lighthouse/claim-detail.json';
+
+describe('A user can navigate to their claim details of a closed claim', () => {
+  it('Shows the correct status for the claim - C30820', () => {
+    const trackClaimsPage = new TrackClaimsPageV2();
+    trackClaimsPage.loadPage(claimsList, claimDetails);
+    trackClaimsPage.verifyInProgressClaim(false);
+    cy.axeCheck();
+  });
+});
+
+describe('A user can navigate to their claim details of an open claim', () => {
+  it('Shows the correct status for the claim - C30820', () => {
+    const trackClaimsPage = new TrackClaimsPageV2();
+    trackClaimsPage.loadPage(claimsList, claimDetailsOpen);
+    trackClaimsPage.verifyInProgressClaim(true);
+    cy.axeCheck();
+  });
+});
+
+describe('A user with more than 3 contentions can click "Show full list" button and see more contentions', () => {
+  it('Has "Show full list" button which reveals more contentions when clicked', () => {
+    const trackClaimsPage = new TrackClaimsPageV2();
+    trackClaimsPage.loadPage(claimsList, claimDetails);
+    trackClaimsPage.verifyInProgressClaim(false);
+    trackClaimsPage.verifyContentions();
+    cy.axeCheck();
+  });
+});
+
+describe('A user that has tracked items can view recent activity', () => {
+  it('Shows recent activity for a user with tracked items', () => {
+    const trackClaimsPage = new TrackClaimsPageV2();
+    trackClaimsPage.loadPage(claimsList, claimDetails);
+    trackClaimsPage.verifyInProgressClaim(false);
+    trackClaimsPage.verifyRecentActivity();
+    cy.axeCheck();
+  });
+});
+
+describe('A user that has more than 10 tracked items gets pagination on recent activity page', () => {
+  it('Shows pagination for a user with more than 10 tracked items', () => {
+    const trackClaimsPage = new TrackClaimsPageV2();
+    trackClaimsPage.loadPage(claimsList, claimDetails);
+    trackClaimsPage.verifyInProgressClaim(false);
+    trackClaimsPage.verifyRecentActivityPagination();
+    cy.axeCheck();
+  });
+});
+
+describe('A user can view primary alert details', () => {
+  it('Shows primary alert details', () => {
+    const trackClaimsPage = new TrackClaimsPageV2();
+    trackClaimsPage.loadPage(claimsList, claimDetailsOpen);
+    trackClaimsPage.verifyInProgressClaim(false);
+    trackClaimsPage.verifyPrimaryAlert();
+    cy.axeCheck();
+  });
+});
+
+describe('A user can view secondary alert details', () => {
+  it('Shows secondary alert details', () => {
```

Comment:
This test is fine but different then the task test that we have noted which is `'If there's a secondary alert, a user can see that alert on the status tab under "Recent Activity" section and click "add it here"' `

I would update the name of this test to say `'A user on a closed claim can view secondary alert details'` and  'Shows closed claim alert details'. 

Then I would add a new test for the secondary alert that should appear under recent activity 
![Screenshot 2024-03-28 at 8 52 52 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/141954992/ab2a5b73-2a6e-4df8-b056-9f70ed7742ab)